### PR TITLE
Update Japanese translation (LINE 163-204)

### PR DESCRIPTION
**Pull request by yosshy**
/hold for japanese developer's review
/assign @t-inu
/cc @atoato88 @s-kawamura-w664

### CODE REVIEW

File: translations/ja.json [hunk 1]
```diff
@@ -160,48 +160,48 @@
...
+	"Docker inside the VM is unavailable. Try running 'minikube delete' to reset the VM.": "VM 内の Docker が利用できません。'minikube delete' を実行して、VM を初期化してみてください。",
+	"Docs have been saved at - {{.path}}": "ドキュメントは次のパスに保存されました - {{.path}}",
 	"Documentation: {{.url}}": "ドキュメント: {{.url}}",
-	"Done! kubectl is now configured to use \"{{.name}}\"": "完了しました！ kubectl が「\"{{.name}}\"」を使用するよう構成されました",
-	"Done! kubectl is now configured to use \"{{.name}}\" cluster and \"{{.ns}}\" namespace by default": "完了しました！ kubectl が「\"{{.name}}\"」クラスタと「\"{{.ns}}\"」ネームスペースを使用するよう構成されました",
-	"Done! minikube is ready without Kubernetes!": "",
-	"Download complete!": "ダウンロードが完了しました",
-	"Downloading Kubernetes {{.version}} preload ...": "Kubernetes {{.version}} のダウンロードの準備をしています",
+	"Done! kubectl is now configured to use \"{{.name}}\"": "終了しました! kubectl が「{{.name}}」を使用するよう設定されました",
+	"Done! kubectl is now configured to use \"{{.name}}\" cluster and \"{{.ns}}\" namespace by default": "終了しました!  kubectl がデフォルトで「{{.name}}」クラスターと「{{.ns}}」ネームスペースを使用するよう設定されました",
```
Comment (t-inu) [inline translations/ja.json:167]:
```suggestion
	"Done! kubectl is now configured to use \"{{.name}}\" cluster and \"{{.ns}}\" namespace by default": "終了しました! kubectl がデフォルトで「{{.name}}」クラスターと「{{.ns}}」ネームスペースを使用するよう設定されました",
```
There is an extra blank.

File: translations/ja.json [hunk 2]
```diff
@@ -160,48 +160,48 @@
...
-	"Ensure you have at least 20GB of free disk space.": "",
-	"Ensure your {{.driver_name}} is running and is healthy.": "",
-	"Environment variables to pass to the Docker daemon. (format: key=value)": "Docker デーモンに渡す環境変数（形式: Key=Value）",
-	"Environment variables to pass to the build. (format: key=value)": "",
-	"Error code docs have been saved at - {{.path}}": "",
-	"Error creating minikube directory": "minikube のディレクトリ作成中にエラーが発生しました",
+	"Ensure that CRI-O is installed and healthy: Run 'sudo systemctl start crio' and 'journalctl -u crio'. Alternatively, use --container-runtime=docker": "CRI-O がインストール済みで正常であることを確認してください: 'sudo systemctl start crio' と 'journalctl -u crio' を実行してください。または、--container-runtime=docker を使用してください",
+	"Ensure that Docker is installed and healthy: Run 'sudo systemctl start docker' and 'journalctl -u docker'. Alternatively, select another value for --driver": "Docker がインストール済みで正常であることを確認してください: 'sudo systemctl start docker' と 'journalctl -u docker' を実行してください。または、--driver に別の値を選択してください",
+	"Ensure that the required 'pids' cgroup is enabled on your host: grep pids /proc/cgroups": "必要な 'pids' がこのホスト上で有効であることを確認してください: grep pids /proc/cgroups",
```
Comment (t-inu) [inline translations/ja.json:190]:
```suggestion
	"Ensure that the required 'pids' cgroup is enabled on your host: grep pids /proc/cgroups": "必要な 'pids' cgroup がこのホスト上で有効であることを確認してください: grep pids /proc/cgroups",
```
It's better not to omit "cgroup".

File: translations/ja.json [hunk 3]
```diff
@@ -160,48 +160,48 @@
...
+	"Ensure that the user listed in /etc/libvirt/qemu.conf has access to your home directory": "/etc/libvirt/qemu.conf にリストされたユーザーがあなたのホームディレクトリーにアクセス可能であることを確認してください",
+	"Ensure that you are a member of the appropriate libvirt group (remember to relogin for group changes to take effect!)": "あなたが適切な libvirt グループメンバーの 1 人であることを確認してください (グループ変更が効果を発揮するための再ログインをお忘れなく)",
```
Comment (t-inu) [inline translations/ja.json:192]:
```suggestion
	"Ensure that you are a member of the appropriate libvirt group (remember to relogin for group changes to take effect!)": "あなたが適切な libvirt グループメンバーの 1 人であることを確認してください (グループ変更を有効にするには再ログインをお忘れなく!)",
```
"有効にするには“ is more easier to understand I think.

**Comment by t-inu** [comment]
@yosshy Since the translation policy was updated to use full-width character for "!", please apply to this PR.

**Comment by atoato88** [comment]
I've checked this PR now.
It's LGTM except for comments from other reviewer.

BTW, do you have plan to replace `!`(hankaku) with `！`(zenkaku) on other PR like #13636?

File: translations/ja.json [hunk 4]
```diff
@@ -160,48 +160,48 @@
...
+	"Ensure that your value for HTTPS_PROXY points to an HTTPS proxy rather than an HTTP proxy": "HTTPS_PROXY の値が HTTP プロキシーではなく HTTPS プロキシーを指定していることを確認してください",
+	"Ensure the tmp directory path is writable to the current user.": "現在のユーザーで tmp ディレクトリーパスが書き込み可能であることを確認してください。",
+	"Ensure you have at least 20GB of free disk space.": "少なくとも 20GB のディスク空き容量があることを確認してください。",
+	"Ensure your {{.driver_name}} is running and is healthy.": "{{.driver_name}} が実行中で正常であることを確認してください。",
+	"Environment variables to pass to the Docker daemon. (format: key=value)": "Docker デーモンに渡す環境変数 (形式: key=value)",
+	"Environment variables to pass to the build. (format: key=value)": "build に渡す環境変数 (形式: key=value)",
```
Comment (s-kawamura-w664) [inline translations/ja.json:198]: If you follow the rules, "。" would be necessary.

```suggestion
	"Environment variables to pass to the Docker daemon. (format: key=value)": "Docker デーモンに渡す環境変数。 (形式: key=value)",
	"Environment variables to pass to the build. (format: key=value)": "build に渡す環境変数。 (形式: key=value)",
```

File: translations/ja.json (hunk 1 above)
Comment (yosshy) [reply translations/ja.json]: done.

File: translations/ja.json (hunk 3 above)
Comment (yosshy) [reply translations/ja.json]: applied.

**Comment by atoato88** [comment]
/lgtm
Now, this PR have been updated that replaced hankaku `!` with zenkaku `！` :smile:

**Comment by t-inu** [comment]
LGTM

**Comment by yosshy** [comment]
@atoato88 @t-inu @s-kawamura-w664 thank you for your reviews.

/unhold
/assign @spowelljr
Japanese review finished. Please merge this.

**Review by spowelljr — APPROVED** [review]

**Pull request merged**